Mathlib/MeasureTheory/Integral/Lebesgue.lean

```diff
@@ -1356,6 +1356,15 @@ theorem MeasurePreserving.set_lintegral_comp_emb {mb : MeasurableSpace β} {ν :
   rw [← hg.set_lintegral_comp_preimage_emb hge, preimage_image_eq _ hge.injective]
 #align measure_theory.measure_preserving.set_lintegral_comp_emb MeasureTheory.MeasurePreserving.set_lintegral_comp_emb
 
+theorem set_lintegral_eq_subtype {s : Set α} (hs : MeasurableSet s) (f : α → ℝ≥0∞) :
+    ∫⁻ x in s, f x ∂μ = ∫⁻ x : s, f x ∂(μ.comap (↑)) := by
+  rw [← (MeasurableEmbedding.subtype_coe hs).lintegral_map, map_comap_subtype_coe hs]
```

Comment:
Should this equality be reversed? The right hand side looks more complicated. I would even be tempted to add a @[simp] attribute to have it simplify to the left hand side (and add simp to the next lemma as well). What do you think?

Reply:
It matches pre-existing `set_integral_eq_subtype`. I'll see where the latter is used and swap sides there too.